integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/validation/WebhookFilterValidator.java

```diff
@@ -0,0 +1,34 @@
+package no.difi.meldingsutveksling.validation;
+
+import lombok.extern.slf4j.Slf4j;
+import no.difi.meldingsutveksling.webhooks.filter.WebhookFilterParser;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+@Slf4j
+public class WebhookFilterValidator implements ConstraintValidator<WebhookFilter, String> {
+
+    @Autowired
+    private WebhookFilterParser webhookFilterParser;
+
+    @Override
+    public void initialize(WebhookFilter constraint) {
+        // NOOP
+    }
+
+    public boolean isValid(String s, ConstraintValidatorContext context) {
+        if (s == null) {
+            return true;
+        }
+
+        try {
+            webhookFilterParser.parse(s).forEach(p -> {
+            });
+            return true;
+        } catch (Exception e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)